app/helpers/document_list_helper.rb

```diff
@@ -49,6 +50,16 @@ def world_location_news_query_params(content_type, slug)
         order: "updated-newest",
         world_locations: [slug],
       }
+    when :publication
+      {
+        content_purpose_supergroup: %w[guidance_and_regulation policy_and_engagement transparency],
+        order: "updated-newest",
```

Comment:
Do we want definitely want this ordering? Latest obvs needs it, but I haven't been sure about the others - I think to me it'd make more sense if they were in order of most viewed (or some other relevance metric). Not sure what the default is

Reply:
I added this as it's the order we're retrieving documents on the World Location News page, so it'll ensure the preview always matches the first two results in the finder.

Reply:
I think we have this issue with other document types - was something I was pondering earlier, as topical event doc types have the same issue. I was thinking the nicer way around it would be for everything everywhere except latest to be ordered by relevance

Reply:
So I guess this is a question that goes beyond just this PR, but wondered what your thoughts were

Reply:
To me, I'd rather the documents were sorted by most recently published, otherwise there's a chance that new documents won't be seen by users of the finder (and the 'most viewed' stats could be skewed towards the older documents, rather than those that are most popular).

If there's something super important, it could be added as a featured document.

Maybe this is something that a PM could make a decision on?

Reply:
OK that makes sense. Will approve this and maybe we can raise the general question with a PM